### ISSUE TITLE
Improve allele autocomplete

### ISSUE THREAD

**Issue by cmungall** [monarch-initiative/biolink-api#327]
For some alleles e.g.
http://www.informatics.jax.org/allele/MGI:6277008

We use the label `Bloc1s6<em1(IMPC)Mbp>`

But the `<` is formatting, a user might do something like

`Bloc1s6em1(IMPC)Mbp`

but this won't return results eg

https://api.monarchinitiative.org/api/search/entity/autocomplete/Bloc1s6%3Cem1(IMPC)Mbp%3E

https://api.monarchinitiative.org/api/search/entity/autocomplete/Bloc1s6%3Cem1
https://api.monarchinitiative.org/api/search/entity/autocomplete/Bloc1s6em1

nsw best approach is to inject synonyms at dipper time, have a post-processing style on all ttls to do this universally, or tweak solr settings...?
